### DISCUSSION TITLE
Latest version 23.9.11

### DISCUSSION THREAD

**Discussion by octavvian** [tbnobody/OpenDTU#1331] [Q&A]
Hi,
have you tried with success to upgrade or fresh install the latest version?
Upgraded OTA from 23.8.28 to 23.9.11 OK, restart no go...
Fresh install generic.factory.bin or ESP32.factory.bin, restart, no AP mode ....
Reinstall 2023.8.28, everything OK.
Any ideas?
 Thanks!

**Comment by avandorp** [comment]
Upgraded directly in the web interface (from the former release). Used the opendtu-generic.bin. Worked flawlessly. On an older ESP32.

**Comment by octavvian** [comment]
Esp board changed, problem solved.
Thanks for confirming.

**Comment by octavvian** [comment]
Closed!
